core/src/main/java/org/stellar/anchor/sep12/Sep12Service.java

```diff
@@ -148,20 +132,14 @@ public void deleteCustomer(Sep10Jwt sep10Jwt, String account, String memo, Strin
     }
 
     boolean existingCustomerMatch = false;
-    for (String customerType : knownTypes) {
-      GetCustomerResponse existingCustomer =
-          customerIntegration.getCustomer(
-              GetCustomerRequest.builder()
-                  .account(account)
-                  .memo(memo)
-                  .memoType(memoType)
-                  .type(customerType)
-                  .build());
-      if (existingCustomer.getId() != null) {
-        existingCustomerMatch = true;
-        customerIntegration.deleteCustomer(existingCustomer.getId());
-      }
+    GetCustomerResponse existingCustomer =
+        customerIntegration.getCustomer(
+            GetCustomerRequest.builder().account(account).memo(memo).memoType(memoType).build());
```

Comment:
This assumes the business server gives unique IDs to account, and memo pairs rather than account, and memo, types. I don't think this assumption is captured in docs. I think we should mention this somewhere.

Reply:
Tracking here: https://stellarorg.atlassian.net/browse/ANCHOR-780